liiatools_pipeline/sensors/location_schedule.py

```diff
@@ -238,4 +261,4 @@ def reports_schedule(context):
                 ),
             )
         else:
-            context.log.info("No new files found, skipping run")
+            context.log.info(f"No new files found with {clean_config}. skipping run")
```

Comment:
Not sure it's necessary to return the whole `clean_config`, the only interesting value from this would be `clean_config.dataset` (everything else is None)